### PR TITLE
OCLOMRS-484: Display mappings on the Q-AND-A pages both on the Create and Edit page

### PR DESCRIPTION
**Pull request by AGMETEOR**
# JIRA TICKET NAME:
[Display mappings on the Q-AND-A pages both on the Create and Edit page](https://issues.openmrs.org/browse/OCLOMRS-484)

# Summary:
Currently, on the page where we either create or edit Q-AND-A concepts, there are no mappings shown. In this ticket, on the page where we create Q-AND-A concepts, fields for adding mappings should be displayed at the bottom of the page. Likewise, on the page where we edit a Q-AND-A concept, mappings should be displayed at the bottom of the page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "uuid": "^3.3.2"
   },
   "scripts": {
-    "start": "chmod +x ./init.sh && ./init.sh && cp env-config.js ./public/ && npm run build:css && react-scripts start",
```
Comment (dkayiwa) [inline package.json]: How is this related to the ticket?

Comment (AGMETEOR) [reply package.json]: @dkayiwa that script stops the application from running and removing it is the only way the application will start.

Comment (dkayiwa) [reply package.json]: Was that caused by changes in this ticket? If not, then it should be a separate ticket and pull request.

Comment (AGMETEOR) [reply package.json]: @dkayiwa no it was not. There's a ticket that's been put up to solve the issue with the start script. Let me rebase.

**Comment by dkayiwa** [comment]
Did you just forget to add tests for these changes?

**Comment by AGMETEOR** [comment]
@dkayiwa when I look at the coverage file, the current tests cover the lines added.
